aws/README.md

```diff
@@ -1,89 +1,177 @@
+
+While creating the access keys, create a policy using the below JSON to limit the access to Cost Explorer service.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": [
+        "ce:DescribeCostCategoryDefinition",
+        "ce:GetRightsizingRecommendation",
+        "ce:GetCostAndUsage",
+        "ce:GetSavingsPlansUtilization",
+        "ce:GetReservationPurchaseRecommendation",
+        "ce:ListCostCategoryDefinitions",
+        "ce:GetCostForecast",
+        "ce:GetReservationUtilization",
+        "ce:GetSavingsPlansPurchaseRecommendation",
+        "ce:GetDimensionValues",
+        "ce:GetSavingsPlansUtilizationDetails",
+        "ce:GetCostAndUsageWithResources",
+        "ce:GetReservationCoverage",
+        "ce:GetSavingsPlansCoverage",
+        "ce:GetTags",
+        "ce:GetUsageForecast"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+The values of `awsCostExplorerAccessKeyId`, `awsCostExplorerSecretAccessKey`, and `awsCostExplorerRegion` should be your Access Key ID, Secret access key and your AWS region code respectively.
+
 ## Commands
 
-- [`ec2_list`](#ec2_list)
-- [`ec2_reboot`](#ec2_reboot)
-- [`ec2_status`](#ec2_status)
-- [`dynamodb_list`](#dynamodb_list)
-- [`rds_list`](#rds_list)
-- [`route53_list`](#route53_list)
-- [`route53_add`](#route53_add)
+- [`awsbill`](#awsbill) - Fetch AWS bill.
+- [`ec2_list`](#ec2_list) - List EC2 instances in the provided region.
+- [`ec2_reboot`](#ec2_reboot) - Reboot an EC2 instance.
+- [`ec2_status`](#ec2_status) - Know the status of EC2 instances.
+- [`dynamodb_list`](#dynamodb_list) - List Database tables.
+- [`rds_list`](#rds_list) - List Amazon RDS (Relational Database Service) instances under your account.
+- [`route53_list`](#route53_list) - List AWS route53 managed hostnames.
+- [`route53_add`](#route53_add) - Add AWS route53 hostname.
 
 ## Usage
 
-### `ec2_list`
+### [`awsbill`](packages/aws/awsbill.js)
+
+Shows your AWS bill.
+
+```sh
+/nc awsbill [<monthYear>] [-task <task>]
+```
+
+To get notified when the bill exceeds a certain amount, follow the instructions below.
+
+1. Create a secret named `awsCostThreshold` with your budget.
+
+   Run `/nc secret_create` and follow the instructions.
+
+   > The value should be a number without your currency unit. ✅ `100` ❌ `$100`
+
+   **Note**: we recommend you to set the budget a bit lesser than your actual budget since Cost Explorer API is delayed by a day.
+
+2. Setup your Slack channel with Commander
+
+   Follow this [guide](https://nimbella.com/resources-commander/guide#create-add-a-channel).
+
+3. Create a Commander task
+
+   - Run `/nc task_create awsbill #general awsbill -task` to create a task.
+
+     Replace `#general` with the slack channel you've added to commander.
+
+   - Run `/nc task_rate awsbill 1 minute` to set your task rate
```

Comment:
there is a cost to making this api call every minute - the docs should warn the user.